genai-perf/README.md

```diff
@@ -74,7 +74,7 @@ The easiest way to install GenAI-Perf is through
 Install the latest release using the following command:
 
 ```bash
-export RELEASE="yy.mm" # e.g. export RELEASE="24.06"
+export RELEASE="yy.mm" # e.g. export RELEASE="24.08"
```

Comment:
The doc templates will handle this going forward.
We should go that route instead of maintaining this every release.

Reply:
discussed offline. updated template accordingly.